common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/BlocksRemoved.java

```diff
@@ -50,7 +50,7 @@ public String toString() {
   public boolean equals(Object other) {
     if (other != null && other instanceof BlocksRemoved) {
       BlocksRemoved o = (BlocksRemoved) other;
-      return Objects.equal(numRemovedBlocks, o.numRemovedBlocks);
+      return numRemovedBlocks == o.numRemovedBlocks;
```

Comment:
@cloud-fan here is one of the change we discussed in https://github.com/apache/spark/pull/24499#discussion_r300459700